app/views/admin/users/show/profile/actions/_banish.html.erb

```diff
@@ -1,20 +1,21 @@
 <div id="banish-for-spam">
   <div class="flex flex-col gap-4">
     <% if @user.comments.where("created_at < ?", 100.days.ago).empty? && @user.created_at < 100.days.ago %>
-      <p><strong>This is extremely destructive.</strong> Banishing will delete all the user's existing content and change their username to <em>@spam_###</em>.</p>
-      <p><strong>Do not do this lightly.</strong></p>
-      <%= form_for(@user, url: banish_admin_user_path(@user), html: { method: :post, onsubmit: "return confirm('Are you sure? This is extremely destructive and irreversible. Banishing will delete all articles and turn their username into @spam_###')", id: nil }) do %>
+      <p>This action is irreversible.</p>
```

Comment:
Reading notes: it looks like we have two chunks of HTML that are identical.  We have chosen to separate them by an `if` then `elsif` to highlight the purpose.

Non-blocking recommendation: make a helper method that answers the question "should we render the banish a user section?"  We could then use line 3 and line 10 as the clauses in our function.

Reply:
I'll refactor this in a follow-up PR, @jeremyf 😊